script/build-and-sign.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+shopt -s nullglob
+set -e
+
+
+for dep in setuptools wheel; do
+    if ! python -m pip -q show "${dep}"; then
+        echo "build: missing dependency '${dep}'" >&2;
+        exit 1;
+    fi
+done
+
+
+KEY_ID=${SIGNING_KEY_ID:-2E390FA0}
+KEY_FILE=${SIGNING_KEY_FILE:-signing.key}
+KEY_FILE_ENC=${KEY_FILE}.enc
+
+version=$(python setup.py --version)
+dist_dir=${STREAMLINK_DIST_DIR:-dist}
+
+wheel_platforms_windows=("win32" "win-amd64")
+
+mkdir -p "${dist_dir}"
+
+echo "build: Building Streamlink sdist" >&2
+python setup.py -q sdist --dist-dir "${dist_dir}"
+
+echo "build: Building Streamlink wheel (universal)" >&2
+python setup.py -q bdist_wheel --dist-dir "${dist_dir}"
+
+for platform in "${wheel_platforms_windows[@]}"; do
+    echo "build: Building Streamlink wheel (${platform})" >&2
+    python setup.py -q bdist_wheel --plat-name "${platform}" --dist-dir "${dist_dir}"
+done
+
+
+if [[ "${CI}" = true ]] || [[ -n "${GITHUB_ACTIONS}" ]]; then
+    echo "build: Decrypting signing key" >&2
+    openssl aes-256-cbc \
+        -K "${RELEASE_SIGN_KEY}" \
+        -iv "${RELEASE_SIGN_IV}" \
+        -in "${KEY_FILE_ENC}" \
+        -out "${KEY_FILE}" \
+        -d
```

Comment:
I have changed the way the file is encrypted, using the [suggested method by GitHub](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#limits-for-secrets). The encrypted file will need to be updated in the repo too. I do not have the original IV and key for the encrypted file as I seem to remember it was done automatically by the travis CLI and I don't think the IV/key were ever exposed to me either. 

```suggestion
	gpg --quiet --batch --yes --decrypt \
		--passphrase="${RELEASE_KEY_PASSPHRASE}" \
		--output "${KEY_FILE}" \
		"${KEY_FILE_ENC}"
```